### PR TITLE
Update SQA documents for changes to MOOSE and PLN4005

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #367)

@cbolisetti You need to go in and add some content the the scs.md and cci.md files.

### CODE REVIEW

**Comment by loganharbour** [comment]
@cbolisetti can you add to scs.md and cci.md in the near future? Mastodon has been missing moose submodule updates for a month because of this.

**Comment by cbolisetti** [comment]
@loganharbour, yes, I am working on this right now. Will get it done this afternoon.

**Comment by cbolisetti** [comment]
@aeslaughter and @loganharbour, just pushed a commit. Please let me know if anything else is needed.

**Comment by loganharbour** [comment]
@aeslaughter wanna look at @cbolisetti's changes?

File: doc/content/sqa/mastodon_cci.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+!template load file=app_cci.md.template app=MASTODON category=mastodon
+
+!template! item key=general-communication
+The best way to contact the MASTODON team is through the [mastodon-users google group](https://groups.google.com/forum/#!forum/mastodon-users).
+
+To join the group, GMail users can just click the "Join group" button.
+Non-GMail users must send an email to:
```
Comment (aeslaughter) [inline doc/content/sqa/mastodon_cci.md:7]: This if fine for now, but you should considering switching to GitHub Discussions.

File: doc/content/sqa/mastodon_sll.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+!template load file=app_sll.md.template app=MASTODON category=mastodon
+
+# BlackBear
+
+Grizzly relies on BlackBear, which is summarized as follows on the project website:
+[github.com/idaholab/blackbear](https://github.com/idaholab/blackbear).
```
Comment (aeslaughter) [inline doc/content/sqa/mastodon_sll.md:6]:
```suggestion
MASTODON relies on BlackBear, which is summarized as follows on the project website:
[github.com/idaholab/blackbear](https://github.com/idaholab/blackbear).
```

**Review by loganharbour — APPROVED** [review]

**Pull request merged**